README.md

```diff
@@ -54,6 +54,7 @@ The configuration file follows a simple structure.
     "showMilliseconds", "True or false, whether the Runtime section should show milliseconds or not.",
     "defaultIDP": "The default value used for IDP when logging in, this IDP can be manually changed in the Web app as well. ",
     "queryFolder": "The base location of the queries, all query locations will start from this folder (relative to public folder.)",
+    "httpProxy": "The http proxy used to execute queries. When left empty, the Comunica query engine will handle it.",
```

Comment:
When using the proxy Comunica will still execute the query, it will only reroute the request via the proxy. This has to be fixed this. Additionally mention that it is mainly done here to deal with CORS headers that are not (correctly) set on the servers that are being queried.

Reply:
I have updated the explanation accordingly in [commit 0e07459](https://github.com/SolidLabResearch/generic-data-viewer/pull/72/commits/0e07459756a7b46c929c9db4e3a653751415a237)